resources/home/dnanexus/utils/query.py

```
@@ -322,16 +324,36 @@ def _find(report):
+        print(f"Encountered {len(errors)} errors during processing:")
+        for i, error in enumerate(errors, 1):
+            print(f"{i}. Report: {error['report']} - Error: {error['error']}")
+
+        # Raise exception with all errors collected
+        raise RuntimeError(f"Failed to process {len(errors)} reports. See logs for details.")
 
     return cnv_data
 
 
-def find_snv_files(reports) -> dict:
+def find_snv_files(reports, build) -> dict:
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)